### DISCUSSION TITLE
Proxy

### DISCUSSION THREAD

**Discussion by V4dimka** [AnonCatalyst/Ominis-OSINT#51] [Q&A]
Hey, everybody. I'm new at this. Can someone help? Please.
![photo_2024-12-11_15-25-14](https://github.com/user-attachments/assets/748ded65-fc16-401d-a7e6-74d4c6df0aa1)

**Comment by VinodHatti-AI-Developer** [comment]
Hi! I can help you with the proxy issue. Based on the error in your screenshot, it appears the tool is having trouble connecting through a proxy server.

Here are some solutions you can try:

1. **Check if the proxy URL is correct**: Make sure your proxy address is properly formatted (e.g., `http://proxy.example.com:8080` or `socks5://proxy.example.com:1080`)

2. **Test proxy connectivity**: Before running the tool, verify that your proxy server is working by testing it with curl:
   ```bash
   curl --proxy http://your-proxy:port https://www.google.com
   ```

3. **Configure environment variables**: Set these environment variables:
   ```bash
   export HTTP_PROXY="http://your-proxy:port"
   export HTTPS_PROXY="http://your-proxy:port"
   ```

4. **Check proxy authentication**: If your proxy requires username/password, format it like: `http://username:password@proxy:port`

5. **Try without proxy**: If you don't actually need a proxy, you can disable it by unsetting the environment variables:
   ```bash
   unset HTTP_PROXY HTTPS_PROXY
   ```

Let me know which approach works for you or if you need more specific help!
